hypothesis-python/tests/numpy/test_gen_data.py

```diff
@@ -1224,7 +1236,10 @@ def test_basic_indices_generate_valid_indexers(
     )
     # Check that disallowed things are indeed absent
     if not allow_newaxis:
-        assert 0 <= len(indexer) <= len(shape) + int(allow_ellipsis)
+        if hasattr(indexer, "__len__"):
```

Comment:
```suggestion
        if isinstance(indexer, tuple):
```

Again, we might as well keep things simple :slightly_smiling_face:

Reply:
Oh yeah 100% agree, don't know why I put that :grimacing: